docs/reference/indices/resolve.asciidoc

```diff
@@ -0,0 +1,108 @@
+[[indices-resolve-index-api]]
+=== Resolve index API
+++++
+<titleabbrev>Resolve index</titleabbrev>
+++++
+
+Resolves the specified name(s) and/or wildcard expression(s) to indices, index
+aliases, and data streams. Multiple expressions and remote clusters are
+supported.
+
+
+[source,console]
+----
+GET /_resolve/index/twitter*
+----
+
+[[resolve-index-api-request]]
+==== {api-request-title}
+
+`GET /_resolve/index/<name>`
+
+
+[[resolve-index-api-path-params]]
+==== {api-path-parms-title}
+
+`<name>`::
++
+--
+(Required, string) Comma-separated name(s) or wildcard expression(s) of the
+indices, index aliases, and data streams to resolve. Resources on
+<<modules-remote-clusters,remote clusters>> can be specified using the
+`<cluster>:<name>` syntax.
+--
+
+[[resolve-index-api-query-params]]
+==== {api-query-parms-title}
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=expand-wildcards]
++
+Defaults to `open`.
+
+[[resolve-index-api-example]]
+==== {api-examples-title}
+
+[[resolve-index-basic-example]]
+===== Basic example
+
+[source,console]
+--------------------------------------------------
+GET /_resolve/index/f*,remoteCluster1:bar*
+--------------------------------------------------
+// TEST[continued]
```

Comment:
Since we're skipping this snippet, we shouldn't need the `// TEST[continued]` comment.
```suggestion
```